# Button mapping
a_button: 0
b_button: 1
x_button: 2
y_button: 3

enable_mov_position: 4
enable_mov_orientation: 5
orientation_button: 9
home_button: 8
